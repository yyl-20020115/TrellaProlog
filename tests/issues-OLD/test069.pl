:- initialization(main).

main :-
	X = (false,1), X.
