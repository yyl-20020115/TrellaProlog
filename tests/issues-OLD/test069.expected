   error(type_error(callable,(false,1)),call/1).
